gradle/license-report-config/allowed-licenses.json

```diff
@@ -66,6 +66,9 @@
     },
     {
       "moduleName": "io.libp2p:jvm-libp2p-minimal"
+    },
+    {
+      "moduleName": "org.jetbrains.kotlinx:kotlinx-coroutines-core"
```

Comment:
are these license additions related to the PR?

Reply:
Yes, it appears to be a new or updated dependency from Tuweni.

I'm going to switch this back to a draft PR though as I've reproduced the performance problem we originally saw in #3419. I can work around the worst of it by removing our use of tuweni-crypto but I think we're going to need a new Tuweni release to fix it properly as things like tuweni-ssz are still using tuweni-crypto and it's now incredibly slow.